### DISCUSSION TITLE
Version 2.5.0

### DISCUSSION THREAD

**Discussion by BourgeoisLab** [kleiner-brauhelfer/kleiner-brauhelfer-2#284] [General]
- Neu: Datenbankversion **2008**
- Neu: Niederländische Übersetzung der Tooltips (vielen Dank dachouffe)
- Neu: Knopf, um Hopfengaben auszugleichen (analog wie Malzgaben)
- Neu: Knopf, um Extraktwerte bei Malz und Zusätze auf 100% zu korrigieren
- Neu: Direkte Druck- und Kopiermöglichkeit für Tabellen.
- Neu: Bemerkungsfelder für Maischen, Kochen, Gärung, Maischplan und Wasseraufbereiung
- Neu: Linke Seite in Brau- und Abfülldaten ersetzt mit Diagrammen
- Neu: "Spickzettel/Zusammenfassung" ersetzt durch "Ausdruck" mit frei konfigurierbare Ansicht
- Neu: Spickzettel überarbeitet (nicht alles rückwärtskompatibel!)
- Neu: Unvergärbare Zusätze
- Neu: Menge der Zusätze für die Gärung inklusive Stopfhofen können sich auf die Anstellmenge beziehen
- Neu: Exportformat Brautomat
- Neu: Tooltips aktualisiert
- Fix: Übermäßige Zeilenumbrüche für die Bemerkungsfelder im Spickzettel
- Fix: Absturz beim Ändern eines Eintrags in der Hauptgärung und Schnellgärprobe

<hr /><em>This discussion was created from the release <a href='https://github.com/kleiner-brauhelfer/kleiner-brauhelfer-2/releases/tag/v2.5.0'>Version 2.5.0</a>.</em>

**Comment by markussoeth** [comment]
Danke für die Arbeit!
